pkg/auth/handler/webapp/csrf_middleware.go

```diff
@@ -115,6 +119,9 @@ func (m *CSRFMiddleware) makeUnauthorizedHandler(secFetchError error) http.Handl
 		// The CSRF protection is not successful but Sec-Fetch-CSRF is.
 		// Record unmatched metrics.
 		if secFetchError == nil {
+			logger := CSRFMiddlewareLogger.GetLogger(ctx)
+			logger.Warn(ctx, "csrf protection failed but sec-fetch-csrf is ok")
```

Comment:
```golang
logger.WithError(cookieError).Warn(
  ctx,
  "mismatched csrf protection result",
  slog.String("cookies_based_status", "error"),
  slog.String("sec_fetch_based_status", "ok"),
)
```

Note that if it is possible, we should to include the cookie-based csrf protection error here. (`cookieError` in my above snippet)

Reply:
Log the `securecookieError` and `csrfFailureReason` instead